README.md

```diff
@@ -77,21 +77,15 @@ psql -h localhost
 
 #### Node Dependencies
 
-Install nvm (node version manager):
-
-```
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash
-```
-__Note:__ additional nvm installation notes found at [the nvm repo](https://github.com/creationix/nvm#install-script).
-
-Install the latest node via nvm:
+Install `nodejs` using `asdf`
 ```
-nvm install 8.7
+asdf plugin-add nodejs https://github.com/asdf-vm/asdf-nodejs.git
 ```
 
-Ensure the latest node is your default node version in new shells:
+Install the project's version
 ```
-nvm alias default 8.7
+asdf nodejs install 8.7.0
+asdf local nodejs 8.7.0 # use asdf global if you want it system-wide
```

Comment:
I thought it could potentially be confusing for some users to have the system default set to this projects node version, so favored to keep it as a local configuration. I am totally open to changing this if anyone thinks there are downsides.

Reply:
Makes sense. Feels kinda dirty that this was our default from the beginning, and that's on me

Reply:
i think we can actually lock in the local `nodejs` version via the `.tool-versions` file, as asdf will read from that!

Reply:
Oh good catch! Will add that.